homeassistant/helpers/reload.py

```diff
@@ -164,7 +164,7 @@ async def async_setup_reload_service(
     async def _reload_config(call: Event) -> None:
         """Reload the platforms."""
         await async_reload_integration_platforms(hass, domain, platforms)
-        hass.bus.async_fire(f"event_{domain}_reloaded", context=call.context)
+        hass.bus.async_fire(f"{domain}_reloaded", context=call.context)
```

Comment:
What impact does this change have?

Reply:
I could not find any references to this event in use. I did find that automation, scene, and template all have reloaded events called `{domain}_reloaded` but I don't believe they support config flows so they wouldn't conflict with any of the config entry reload events that this change impacts

Reply:
I reverted this change based on @bdraco 's comment and increased the limit to 64